GameServer/Extensions.cs

```diff
@@ -208,5 +209,10 @@ public static TeamId GetEnemyTeam(TeamId team)
 
             return dic[team];
         }
+
+        public static ClientInfo GetClientInfoByChampion(Champion champ)
```

Comment:
This can't exist here, sorry. It's a static function depending on non-static objects that are being made static. I'd rather recommend you to add this function to the player manager directly.